2/Dockerfile.rhel7

```diff
@@ -46,7 +46,8 @@ EXPOSE 8080 50000
 # /usr/lib64/jenkins will subsequently get redirected to /usr/lib/jenkins; it is confirmed that the 3.7 jenkins RHEL images 
 # do *NOT* have a /usr/lib64/jenkins path
 RUN ln -s /usr/lib/jenkins /usr/lib64/jenkins && \
-    INSTALL_PKGS="dejavu-sans-fonts wget rsync gettext git tar zip unzip openssl bzip2 dumb-init  java-1.8.0-openjdk java-1.8.0-openjdk-devel" && \
+    JDK_VER=$(if [ "$(uname -m)" == "s390x" ]; then echo -n 11; else echo -n 1.8.0; fi) && \
```

Comment:
see https://github.com/openshift/jenkins/blob/master/2/Dockerfile.rhel7 to calc the implementation.